docs/extensions/tutorials/_posts/1970-01-13-UsingLocalization.md

```diff
@@ -0,0 +1,211 @@
+---
+layout: doc
+permalink: /docs/extensions/tutorials/using-localization
+title: Localization
+section: Tutorials
+---
+
+# Localization
+
+i18n stands for internationalization (i, 18 letters, n). Shoutem's localization is based on [i18n-js by fnando](https://github.com/fnando/i18n-js). Each extension has it's own set of translations located in the `app` segment in the `translations` directory. These extension-level translations are fallbacks for when a new extension is made and an existing language file doesn't have it's strings.
+
+Language files are uploaded by the app owner using the `shoutem.i18n` settings page.
+
+<p class="image">
+<img src='{{ site.url }}/img/tutorials/localization/i18n-settings-page.png'/>
+</p>
+
+## Using your Translation
+
+To use your translation, you can download and unzip [this file]({{ site.url }}/static/localization/en.json.zip) and replace all the English strings with anything you like. Either a different language or alternative English strings.
```

Comment:
`...replace all the English strings with your own, either a different language or alternative English strings.`

Reply:
A lot of people will come here just wanting to find out how to translate the whole app to their language. Where can we explain this use case? Perhaps somewhere here in separate section or create a smaller but more "focsed" article?